foolbox/attacks/base.py

```diff
@@ -1,5 +1,8 @@
 import logging
-from abc import ABC, abstractmethod
+from abc import ABCMeta, abstractmethod
+
+# compatible with Python 2 *and* 3:
+ABC = ABCMeta('ABC', (object,), {'__slots__': ()})
```

Comment:
I would prefer to use this code only for old Python versions and use the original code for Python 3.4 and newer, see: `https://stackoverflow.com/a/41622155/8178355`